packages/ios-client/Sources/FishjamClient/media/Dimensions.swift

```diff
@@ -9,12 +9,6 @@ extension Dimensions {
     public static let aspect4By3 = 4.0 / 3.0
 }
 
-extension Dimensions: Equatable {
-    public static func == (lhs: Dimensions, rhs: Dimensions) -> Bool {
```

Comment:
**suggestion:** 
Maybe instead od overloading equal operator write method named `equal` ?

Reply:
Then it could be easier to use in future

Reply:
I thought about it but I decided against it. While it's true that it might be used in the future, this future might as well never come so why keep additional code just in case. Also naming function `equal` might be confusing if there could be an operator function.

However now that I think of it, it will be better if I just wrap this type in our own type instead of typealiasing